source/features/clean-repo-sidebar.tsx

```diff
@@ -0,0 +1,31 @@
+import './clean-repo-sidebar.css';
+
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	// Hide empty meta if it’s not editable by the current user
+	if (!select.exists('.repository-content [aria-label="Edit repository metadata"]')) {
+		select('.repository-content .BorderGrid-cell:first-child > .text-italic')?.remove();
+	}
+
+	// Hide redundant "Releases" section
+	const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]');
+	sidebarReleases!.closest('.BorderGrid-row')!.setAttribute('hidden', '');
```

Comment:
This should work, too:

```suggestion
	sidebarReleases!.closest('.BorderGrid-row')!.hidden = true;
```

See https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden